src/SFA.DAS.FE-Choices.Database/StoredProcedures/GetLatestEmployerSatisfaction.sql

```diff
@@ -3,4 +3,4 @@ AS
 
 SELECT  [UKPRN] FinalScore, [Employers] AS TotalCount, [Responses] AS ResponseCount
 FROM [dbo].[EmployerSatisfaction]
-WHERE [Year] = (SELECT MAX([Year]) FROM [dbo].[EmployerSatisfaction])
\ No newline at end of file
+WHERE [HybridYear] = (SELECT MAX([HybridYear]) FROM [dbo].[EmployerSatisfaction])
```

Comment:
the code for this stored procedure is more efficient if you used this:

CREATE PROCEDURE [dbo].[GetLatestEmployerSatisfaction]
AS
SET NOCOUNT ON;
Declare @maxhybrid varchar(max)

set @maxHybrid = SELECT MAX([HybridYear]) FROM [dbo].[EmployerSatisfaction]
SELECT  [UKPRN] FinalScore, [Employers] AS TotalCount, [Responses] AS ResponseCount
FROM [dbo].[EmployerSatisfaction]
WHERE [HybridYear] = @maxHybrid